### PR TITLE
Depend on current version of System.NetSecurity

### LINKED ISSUE #1262: All TCP ExplicitUserNameAndPassword tests are failing
They are failing because of  https://github.com/dotnet/corefx/issues/9160.

### PR DESCRIPTION
**Pull request by hongdai**
* We need the fix for issue https://github.com/dotnet/corefx/issues/9160
* I have verified manually we can now set Explict username/pwd for TCP.

fixes #1262

### CODE REVIEW

File: src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/project.json [hunk 1]
```diff
@@ -15,10 +15,16 @@
...
+      "imports": [
+        "dotnet5.4"
+      ],
+    "dependencies": {
+        "System.Net.Security": "4.0.1-beta-24322-03",
+     },
+   },
   },
   "supports": {
-    "coreFx.Test.netcore50": {},
```
Comment (hongdai) [inline src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/project.json]: @StephenBonikowsky @roncain I have to remove this line as this line does package validation for projectN and it fails for System.Net.Security as it does not support projectN. I'm going to validate this change both on ProjectN from projectK and ProjectN using Helix. If they both pass, I think the check here is unnecessary.

**Comment by hongdai** [comment]
Hold on the review, it does not work on Linux platform, I need to figure out why.

File: src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/project.json (hunk 1 above)
Comment (StephenBonikowsky) [inline src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/project.json]: I remember seeing that before, makes sense.

**Comment by iamjasonp** [comment]
You might need to add the reference to test-runtime/project.json as well :)

**Comment by hongdai** [comment]
@iamjasonp Yeap, the only way to make it work without updating all test Jason file is to add it to test.runtime.

**Comment by roncain** [comment]
LGTM.
Does it build and run in Net Native? I see #1235 will keep it from running the test but it would be good to confirm other security test run okay. It'd be enough to run against UWP locally (no need for TFS)

**Comment by iamjasonp** [comment]
LGTM

**Comment by hongdai** [comment]
I have verified both ProjectN from ProjectK run and .Net native, no regression found.

**Pull request merged**